resolver/resolver.cc

```diff
@@ -448,6 +484,11 @@ class ResolveConstantsWalk {
             sym = cls.data(gs)->owner;
         }
 
+        // Explicitly consider an empty top-level scope as one to skip.
```

Comment:
WDYT about adding a comment here about how this comes up?  (Specifically in the context of test packages, but maybe packages in general--the former because we didn't have a case like this prior to this PR, but the latter if you thought of an interesting case where this would be true for non-test packages.)

Reply:
I think that's a good idea. I'll add some more text here.